packages/ckeditor5-dev-ci/bin/notifycirclestatus.js

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env node
+
+/**
+ * @license Copyright (c) 2003-2023, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/* eslint-env node */
+
+const allowedBranches = require( '../lib/data/allowedBranches.json' );
+const formatMessage = require( '../lib/formatMessage' );
+const printLog = require( '../lib/printLog' );
+const slackNotify = require( 'slack-notify' );
+
+const ALLOWED_EVENTS = [
+	'api',
+	'webhook'
+];
+
+// This script assumes that is being executed on Circle CI.
+// Step it is used on should have set value: `when: on_fail`, since it does not
+// check whether the build failed when determining whether the notification should be sent.
+// Described environment variables starting with "CKE5" should be added by the integrator.
+
+const {
+	/**
+	 * @enum {String} Token to a Github account with the scope: "repos". It is required for obtaining an author of
+	 * the commit if the build failed. The repository can be private and we can't use the public API.
+	 */
+	CKE5_GITHUB_TOKEN,
```

Comment:
```suggestion
	GITHUB_TOKEN,
```

Reply:
In my opinion, adding `CKE5_` prefix to environment variables explicitly shows that they are defined by us, for our scripts. In variables such as `CIRCLE_TOKEN` that might not be obvious at a first glance. I propose that we adopt this prefix as our convention in naming our CI environment variables. I left the old names in Travis script in order not to introduce a breaking change, but since this is a new script, we are free to name the variables in any way.

Reply:
☝️ After internal discussion, we agreed to this.